### DISCUSSION TITLE
Instant translate using API

### DISCUSSION THREAD

**Discussion by leandrofeder** [tolgee/tolgee-platform#2519] [Q&A]
Is there any way to send a sentence and return the translation in the languages I want via API?

**Comment by JanCizmar** [comment]
Hello! Thanks for the query. Tolgee is not a translation service. It's a tool for managing localization strings. For this, you should use APIs from DeepL, Google, etc.

In theory, you can use the suggestion endpoint. But really, Tolgee is designed for something other than this. 

https://tolgee.io/api/suggest-machine-translations
